.github/workflows/hermes.yaml

```diff
@@ -47,7 +47,16 @@ jobs:
       - name: Build Cairo contracts
         working-directory: ./cairo-contracts
         run: |
-          scarb build -p starknet_ibc_contracts
+          scarb --profile release build -p starknet_ibc_contracts
```

Comment:
using `--profile release` to reduce contract class size.

IBC Core contract class with `dev` and `release` profiles are 2.5M and 2M respectively.